pyfar/plot/_line.py

```diff
@@ -28,6 +28,19 @@ def _time(signal, dB=False, log_prefix=20, log_reference=1, unit="s",
         ymin = ymax - 100
     else:
         data = signal.time.T
+    # get data defined in 'show_real_imag_abs'
+    if show_real_imag_abs == 'real':
+        data = np.real(data)
+        y_label = 'Amplitude'
+    elif show_real_imag_abs == 'imag':
+        data = np.imag(data)
+        y_label = 'Amplitude (imaginary)'
+    elif show_real_imag_abs == 'abs':
+        data = np.abs(data)
+        y_label = 'Amplitude (absolute)'
+    else:
+        raise ValueError('`show_real_imag_abs` has to be `real`, `imag`, or '
+                         f'`abs`, but is {show_real_imag_abs}.')
     # auto detect the time unit
```

Comment:
- Could probably go into a private function to be reused.
- And you made it very clear, that we need a better name for the parameter :)